ui/00_message/tutorial_guide/steps/tutorial_guide_step24.xml

```diff
@@ -20,5 +26,14 @@
 　【&lt;KC UI_Y&gt;】透明度の変更
 ＜フリーマーカー＞　
 　【&lt;KC UI_MAP_FREEMARKER&gt;】設置／消去（長押し）</original>
+    <translation>Operating the "Map" in the Game Menu.
+
+(Switch between Detail/Field/Region)
```

Comment:
I don't think () is a good analogue for ＜＞ in this case.

Perhaps just like: `"Switch between Detail/Field/Region"` or `Switch between Detail/Field/Region:`?

Beside that, it should probably be `Switching between Detail/Field/Region` or `Detail/Field/Region Switch`.